style.css

```diff
@@ -117,10 +117,10 @@ margin: auto;
 .locationbackground {
 width: 1200px;
 height: 500px;
+background-image: url(./images/img-locations-background.jpeg);
```

Comment:
the effect that you're trying to achieve is usually done with the `background-image` property 🙂 
https://developer.mozilla.org/en-US/docs/Web/CSS/background-image